### ISSUE TITLE
tesseract4.0 in the Centos7 virtual machine on the server machine is very slow

### ISSUE THREAD

**Issue by banbsyip** [tesseract-ocr/tesseract#1910]
I fist tested my project in Centos7 virtual machine on my laptop(environment: win10,64bit) with tesseract4.0 to identification number from the pictures, the pictures are 14kb-40kb, and the run time was just around 3s-5s. 
At the same time i tested the project in Centos7 virtual machine on the server machine with tesseract3.0 costed 2s-3s .
But when i tested the project in Centos7 virtual machine on the server machine,it costed 30s-40s, the slowest time was 125 seconds. I am very confused about this now. I don't know why this happened. Has anyone encountered this situation?

**Comment by banbsyip** [comment]
i used tesseract3.0 with eng.traineddata(tessdata),tesseract4.0 with eng.traineddata(tessdata_best)

**Comment by amitdo** [comment]
1) What's the output of `tesseract -v` in each case?
2) Check how many cpu cores are used during tesseract execution
3) Use fast / traineddata repo

**Comment by banbsyip** [comment]
> 1. What's the output of `tesseract -v` in each case?
> 2. Check how many cpu cores are used during tesseract execution
> 3. Use fast / traineddata repo

I collected a detail testdata as follow:
I made 3 test pictures,named p1(11.5kb),p2(33.3kb) and p3(11.0kb).each picture test 3 times(I can't upload the pictures). In my project i used 7 processes to run the program for identifying pictures,the code as follow:
p = Pool(7)
    for i in range(7):
        imeiStr.append(p.apply_async(Org_ocr, args=(i, img)))#Org_ocr : the function to identify pictures

case 1: 
cpu cores:  6
enviroment: 
virtual machine centos7(3.10.0-862.11.6.el7.x86_64) on server machine, 
Python 3.5.2 :: Anaconda 4.2.0 (64-bit)
tesseract 3.04.02dev
 leptonica-1.72
  libjpeg 6b (libjpeg-turbo 1.2.90) : libpng 1.5.13 : libtiff 4.0.3 : zlib 1.2.7

tessdata:eng.traineddata(22M)
                             test1       test2      test3
p1' test time:         2.02s      1.90s      2.06s
p2' test time:         2.27s      2.28s      3.67s
p3' test time:         1.24s      1.24s      1.25s

case 2:
cpu cores: 4
enviroment: 
virtual machine centos7(3.10.0-862.11.6.el7.x86_64) on server machine, 
Python 3.6.5 :: Anaconda, Inc.
tesseract 4.0.0-beta.4-140-g59e4
 leptonica-1.74.4
  libjpeg 6b (libjpeg-turbo 1.2.90) : libpng 1.5.13 : libtiff 4.0.3 : zlib 1.2.7
 Found AVX
 Found SSE

tessdata:eng.traineddata(22M)
                             test1       test2      test3
p1' test time:        55.67s     66.61s   61.73s
p2' test time:        69.85s     79.19s   83.47s
p3' test time:        57.00s     59.91s   56.19s

tessdata_best:eng.traineddata(14.7M)
                             test1       test2      test3
p1' test time:        65.51s     63.67s   69.00s
p2' test time:        63.94s     89.63s   83.92s
p3' test time:        64.72s     54.93s   53.81s

the tessdata_fast'time is similar with tessdata_best, i do not give the test data here.
when i tested ,i found the cost of the cpu nearly 100%, after i changed cpu cores from 4 to 8, the test time is similar with case 1.

case 3:
enviroment: 
win10 i7-8550U cpu, 1.80GHz 2.00GHz, 8GB RAM
(yi) C:\Users\19133\Desktop\qr_pic\picture_ocr_1>    Python 3.7.0
tesseract v4.0.0-beta.1.20180608
 leptonica-1.76.0
  libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 1.5.3) : libpng 1.6.34 : libtiff 4.0.9 : zlib 1.2.11 : libwebp 0.6.1 : libopenjp2 2.2.0

tessdata:eng.traineddata(22M)
                             test1       test2      test3
p1' test time:         2.87s      2.76s      2.76s
p2' test time:         4.42s      3.87s      4.30s
p3' test time:         2.92s      2.65s      2.57s

case 4:
enviroment: 
virtual machine centos7(3.10.0-514.el7.x86_64) on win10(i7-7500U CPU @2.7GHz 2.90GHz.16GB RAM), 
Python 3.6.1 :: Anaconda 4.4.0 (64-bit)
tesseract 4.0.0-beta.4-138-g2093
 leptonica-1.76.0
  libjpeg 6b (libjpeg-turbo 1.2.90) : libpng 1.5.13 : libtiff 4.0.3 : zlib 1.2.7
 Found AVX2
 Found AVX
 Found SSE

tessdata:eng.traineddata(22M)
                             test1       test2      test3
p1' test time:         6.84s      5.44s      5.42s
p2' test time:       14.72s      12.02s    12.31s
p3' test time:         8.20s      5.79s      5.74s

tessdata_best:eng.traineddata(14.7M)
                             test1       test2      test3
p1' test time:        10.10s     10.48s    8.32s
p2' test time:       17.29s      17.17s    14.35s
p3' test time:         7.94s      7.09s      7.60s

all the test data is here,the test time of case 2 is unbearable, after i changed the cpu cores from 4 to 8, the test time stabilized in about two seconds. I'm very thankful for your suggestion, thank you very much!
